src/test/java/teammates/logic/core/FeedbackResponseCommentsLogicTest.java

```diff
@@ -346,6 +353,148 @@ public void testGetFeedbackResponseCommentsForQuestionInSection_withSectionName_
         assertEquals(0, comments.size());
     }
 
+    @Test
+    public void testIsResponseCommentVisibleForUser() {
+        StudentAttributes student = dataBundle.students.get("student1InCourse1");
+        CourseRoster roster = new CourseRoster(new ArrayList<>(dataBundle.students.values()),
+                new ArrayList<>(dataBundle.instructors.values()));
+        Set<String> studentsEmailInTeam = new HashSet<>();
+        List<StudentAttributes> studentsInTeam =
+                roster.getTeamToMembersTable().getOrDefault(student.getTeam(), Collections.emptyList());
+        for (StudentAttributes teammates : studentsInTeam) {
+            studentsEmailInTeam.add(teammates.getEmail());
+        }
```

Comment:
Consider using stream here? That is: after getting a stream of student in team, collect it into a set.

Reply:
Thank you for your suggestion!